main.go

```diff
@@ -77,16 +97,11 @@ func main() {
 				} else if c.String("consumer") == "title" {
 					p.consumer = &TitleConsumer{out: os.Stdout}
 				} else {
-					url := c.String("url")
-					index := c.String("index")
-
-					client, err := elastic.NewClient(
-						elastic.SetURL(url),
-						elastic.SetSniff(false))
+					client, err := esClient(url, index, v4)
 					if err != nil {
 						return err
 					}
-					es, err := client.BulkProcessor().Name("MyBackgroundWorker-1").Do(context.Background())
```

Comment:
oh come on, the old name was way better.

Reply:
I should have changed it to `MyBackgroundWorker-6`.